### ISSUE TITLE
Focus is not visible over linking banner

### ISSUE THREAD

**Issue by wfenrich** [CeON/fairchive#2323]
When I tab through the clickable elements of the page, the focus on the linking banner is not visible. The WCAG states that it should be – the user has no idea where s/he is and what will be "clicked" if s/he presses "Enter". 

How to set up a banner?

1. Text messages and banner should be enabled by a superuser in "General Information" of given dataverse.
![image](https://github.com/user-attachments/assets/b75c9469-1b36-4626-a0d3-1381ab7d3d7f)

2. On a dataverse page: Edit -> Messages + Banners -> Banners (Tab)
